### PR TITLE
Update asset fetch script to use real repo

### PR DESCRIPTION
**Pull request by jelbourn**

### CODE REVIEW

File: tools/fetch-assets.sh [hunk 1]
```diff
@@ -3,31 +3,36 @@
...
+
+# Create folders into which to copy content and assets.
+mkdir -p $tmpAssetClonePath
+mkdir -p $exampleAssetsPath $docAssetsPath
 
 # Pull assets from repo and remove .git directory
-git clone $DOCS_REPO $DEST_PATH
-rm -rf $DEST_PATH/.git
+git clone $docsContentRepo $tmpAssetClonePath
+rm -rf $tmpAssetClonePath/.git
```
Comment (DevVersion) [inline tools/fetch-assets.sh]: Why do we need to remove the `.git/` folder?

Comment (jelbourn) [inline tools/fetch-assets.sh]: No idea- @riavalon what was the reason for that?

Comment (riavalon) [inline tools/fetch-assets.sh]: Honestly, I'm not quite sure anymore. It doesn't seem like it's needed, though, so it should be removed.

Comment (DevVersion) [inline tools/fetch-assets.sh]: We could remove it I think. Other than that this PR should be ready?

**Pull request merged**